docs/requirements/find_audit_markers

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# This method finds all instances of each UUID in the codebase.
+#
+
+audit_dir=$(cd $(dirname ${0}); pwd)
+root=$(dirname $(dirname ${audit_dir}))
+cat ${audit_dir}/audit_events.csv | tail +2 | while read r; do
```

Comment:
`tail +2` actually gives an error message for me, with `tail` from GNU coreutils 8.23. I was surprised, because I thought that used to work. *shrug*.